### ISSUE TITLE
Help! There is no colour change

### ISSUE THREAD

**Issue by cup-kakes12** [CowCoding0/Gartic-Phone-Draw-Bot#5]
When the colour test is run it shows all the colours however when i try to launch it, it only draws in the colour that was selected last. It does not change colours!

**Comment by VicDropfr** [comment]
Same issue here, using last version

**Comment by CowCoding0** [comment]
Areyo

**Comment by CowCoding0** [comment]
Are you sure you're saving the color positions

**Comment by VicDropfr** [comment]
yes because when i load the color scheme and run the color test it displays all the colors

**Comment by CowCoding0** [comment]
Hmm do you have all the files in the same directory

**Comment by VicDropfr** [comment]
> Hmm do you have all the files in the same directory

yes, in a folder
